src/views/evaluation/Evaluation.js

```diff
@@ -112,11 +102,26 @@ window.onload = function(){
                     tHeader : ["Nume", "Tehnologie", "Nivel"],
                     ImageLink: "/assets/button.jpg",
                     }
-    footerContent = {Footer : "SoftVision@2017"}
-    headerContent = {Logo : "/assets/logo-v2.jpg",
-                    Link: ["Evaluationv2.html", "NewEvaluationv2.html", "Loginv2.html"],
-                    Text: ["Evaluation", "New Evaluation", "Logout"]}
-    option = {tContent : tableContent, fContent : footerContent, hContent : headerContent}
+    __footerContent = {Footer : "SoftVision@2017"}
+
+    __headerContent = [{
+        Link: "Evaluationv2.html",
+        Text: "Evaluation"
+    },
+    {
+        Link: "NewEvaluationv2.html",
+        Text: "New Evaluation"
+    },
+    {
+        Link: "Loginv2.html",
+        Text: "Logout"
+    }]
+
+
+    __logoContent = {Link : "assets/logo-v2.jpg",
+    Alt: "Company logo"}
+
+    option = {tContent : tableContent, footerContent : __footerContent, logoContent : __logoContent, headerContent : __headerContent}
```

Comment:
By the way, about the discussion we were having earlier today regarding variable vs object property naming, here's something pretty awesome that ES6 (which we'll hopefully get to discuss in more detail as this training progresses) knows how to do in a much simpler way: if you already have a bunch of variables called `tableContent` / `footerContent` / `logoContent` / `headerContent`, you can create an object bundling all of these up like so:
`options = { tableContent, footerContent, logoContent, headerContent };`. JavaScript will then infer the name of the properties you wanted from the names of of each of the variables, making this equivalent to `options = { tableContent: tableContent, footerContent: footerContent, logoContent: logoContent, headerContent: headerContent };`, but in less words.